robot_smach_states/examples/manipulation/example_set_gripper.py

```diff
@@ -0,0 +1,28 @@
+#ROS
+import rospy
+import argparse
+
+# TU/e Robotics
+from robot_skills.get_robot import get_robot
+from robot_skills.arms import GripperState
+
+# Robot Smach States
+from robot_smach_states.manipulation import SetGripper
+import robot_smach_states.util.designators as ds
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description="Test the gripper state whether it can open and close")
+    parser.add_argument("--robot", default="hero", help="Robot name (amigo, hero, sergio)")
+    args = parser.parse_args()
+
+    rospy.init_node("Set_gripper_state")
+    robot = get_robot(args.robot)
+    arm = ds.UnoccupiedArmDesignator(robot, {})
+
+    for gripper_state in [GripperState.CLOSE, GripperState.OPEN]:
+        set_gripper_state = SetGripper(robot, arm, gripperstate=gripper_state)
+        outcome = set_gripper_state.execute()
+        rospy.loginfo("Set gripper state to {}, outcome: {}".format(gripper_state, outcome))
+        rospy.sleep(2.5)
```

Comment:
Why a sleep?

Reply:
It is done so that the gripper does not immediately opens after it closes. This makes it easier to see that it has done the action.